.github/workflows/container-validation-dynamo.yml

```
@@ -79,6 +79,16 @@ jobs:
           if-no-files-found: error
           path: |
             ${{ env.PYTEST_XML_FILE }}
+      - name: Collect and Print GitHub Actions Metrics
+        if: always()  # Run even if previous steps fail
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          # Install required Python packages
+          pip install pandas requests
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)